docs/jupyter/geometry/ray_casting.ipynb

```diff
@@ -3,6 +3,11 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # Press Ctrl/Cmd-C in the visualization window to copy the current viewpoint
</pre><p>This comment is meant for people writing the tutorial notebooks, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D/pull/4556/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Also for users, since they may wonder if the author arrived at these precise viewpoint numbers by trial and error. It's not related to this tutorial, but I think its useful in general since its a pretty convenient feature for comparing different results, etc. [We should add it to the new filament GUI as well]</p>